### PR TITLE
Services Guides Refresh: SRCS and CRR

### PR DESCRIPTION
**Pull request by hasheddan**
### Description of your changes

Updates the services guides to use SRCS and CRR.

TODO:
- [x] Update AWS Services Guide
- [x] Test AWS Services Guide
- [x] Update GCP Services Guide
- [x] Test GCP Services Guide
- [x] Update Azure Services Guide
- [ ] Test Azure Services Guide

### Checklist

I have:
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [x] Ensured this PR contains a neat, self documenting set of commits.
- [x] Updated any relevant [documentation], [examples], or [release notes].
- [x] Updated the RBAC permissions in [`clusterrole.yaml`] to include any new types.

[documentation]: https://github.com/crossplaneio/crossplane/tree/master/docs
[examples]: https://github.com/crossplaneio/crossplane/tree/master/cluster/examples
[release notes]: https://github.com/crossplaneio/crossplane/tree/master/PendingReleaseNotes.md
[`clusterrole.yaml`]: https://github.com/crossplaneio/crossplane/blob/master/cluster/charts/crossplane/templates/clusterrole.yaml

### CODE REVIEW

File: docs/services/aws-services-guide.md [hunk 1]
```diff
@@ -430,74 +118,119 @@ kubectl apply -f stack-aws.yaml
...
+        - cidrIp: 0.0.0.0/0
+          description: all ips
+  providerRef:
+    name: aws-provider
+EOF
+
+kubectl apply -f aws-sg.yaml
+```
 
 #### Cloud-Specific Resource Classes
```
[the remark this answers is not in the document]
Comment (hasheddan) [reply docs/services/aws-services-guide.md]: Good call!

File: docs/services/aws-services-guide.md [hunk 2]
```diff
@@ -1,8 +1,5 @@
 ---
-title: Using AWS Services
-toc: true
-weight: 430
-indent: true
+title: Using AWS Services toc: true weight: 430 indent: true
```
Comment (negz) [inline docs/services/aws-services-guide.md]: Looks like this could be the product of a rogue reformat?

File: docs/services/aws-services-guide.md [hunk 3]
```diff
@@ -34,154 +29,25 @@ local machine.
...
-```
-
-Once the provisioning is completed, the above command will return `ACTIVE`.
-
 #### Configuring `kubectl` to communicate with the EKS cluster
 
-Once the cluster is created and is `ACTIVE`, we configure `kubectl` to target
-this cluster:
+In order to communicate with with your EKS cluster, you must configure
+`kubectl`.
```
Comment (negz) [inline docs/services/aws-services-guide.md]: Given that we're (thankfully) removing the preface in which we teach folks how to setup EKS, do we still feel like it's useful to teach folks how to setup `kubectl`? I suggest we remove this and include a link to the EKS docs.

File: docs/services/aws-services-guide.md [hunk 4]
```diff
@@ -430,74 +118,119 @@ kubectl apply -f stack-aws.yaml
...
+kind: DBSubnetGroup
+metadata:
+  name: sample-dbsubnetgroup
+spec:
+  groupName: sample_dbsubnetgroup
+  description: EKS vpc to rds
+  subnetIds:
+    - ${SUBNET_ONE_ID}
+    - ${SUBNET_TWO_ID}
+    - ${SUBNET_THREE_ID}
```
Comment (negz) [inline docs/services/aws-services-guide.md]: I know this was the existing convention, but personally I feel like the bashisms intended to make it possible to copy and run this guide make it seem more intimidating than it needs to be, and could be mistaken by a casual observer as indicating that you _have to_ use bash scripting to build Crossplane YAML. Given we can now use cross-resource-references my preference would be to drop all the `cat >` and `${VARIABLES}` and just repeat things like example subnet names throughout our examples.

File: docs/services/aws-services-guide.md [hunk 5]
```diff
@@ -506,212 +239,101 @@ that we configured earlier.
...
-classes for other non-AWS providers, and reference them for a class of service
-in the `app-project1-dev` namespace.
+Resource claims are used to create external resources by being scheduled to a
+resource class and creating new managed resource or binding to an existing
+managed resource directly. This can be accomplished in a variety of ways
+including referencing the class or managed resource directly, providing labels
+that are used to match to a class, or by defaulting to a class that is annotated
+with `resourceclass.crossplane.io/is-default-class: "true"`. In the
+`RDSInstanceClass` above, we added the default annotation, so our claim will
+bind to it automatically if no other classes exist with annotation. If there are
```
Comment (negz) [inline docs/services/aws-services-guide.md]:
```suggestion
bind to it automatically if no other classes exist with said annotation. If there are
```

File: docs/services/aws-services-guide.md [hunk 6]
```diff
@@ -506,212 +239,101 @@ that we configured earlier.
...
 indicates the managed resource was successfully provisioned and is ready for
 consumption. You can see when claim is bound using the following:
 
 ```bash
-$ kubectl get mysqlinstances -n app-project1-dev
-NAME          STATUS   CLASS            VERSION   AGE
-mysql-claim   Bound    mysql-standard   5.6       11m
+$ kubectl get mysqlinstances
+NAME          STATUS   CLASS-KIND         CLASS-NAME       RESOURCE-KIND   RESOURCE-NAME               AGE
+mysql-claim            RDSInstanceClass   standard-mysql   RDSInstance     default-mysql-claim-5p66w   9s
```
Comment (negz) [inline docs/services/aws-services-guide.md]: The paragraph leading into this example implies we probably want it to say `Bound` under the `STATUS` column.

File: docs/services/aws-services-guide.md [hunk 7]
```diff
@@ -815,76 +437,18 @@ becomes available, then navigate to the address. You should see the following:
...
-running, execute the following commands:
+Because we put all of our configuration in a single directory, we can delete it all with this command:
 
 ```bash
-kubectl delete -f mysql-claim.yaml
-kubectl delete -f mysql-standard.yaml
-kubectl delete -f aws-mysql-standard.yaml
-kubectl delete -f app-project1-dev-namespace.yaml
-kubectl delete -f aws-infra-dev-namespace.yaml
+kubectl delete -f wordpress/
```
Comment (negz) [inline docs/services/aws-services-guide.md]: Does this need to be `-Rf`?

File: docs/services/aws-services-guide.md [hunk 8]
```diff
@@ -895,28 +459,20 @@ In this guide we:
 * Configured RDS to communicate with EKS
```
Comment (negz) [inline docs/services/aws-services-guide.md]: Should we keep the fact that we setup an EKS cluster in this list?

File: docs/services/aws-services-guide.md [hunk 9]
```diff
@@ -34,154 +29,25 @@ local machine.
 ## Preparation
```
Comment (negz) [inline docs/services/aws-services-guide.md]: ^ As best as I can tell `jq` and the AWS CLI are no longer prereqs for this guide. 🎉

File: docs/services/aws-services-guide.md [hunk 10]
```diff
@@ -895,28 +459,20 @@ In this guide we:
...
-* Created an infrastructure (`aws-infra-dev`) and application
-  (`app-project1-dev`) namespace
 * Setup an AWS `Provider` with our account
-* Created a `RDSInstanceClass` in the ` with configuration using RDS configuration earlier
-* Created a `MySQLInstanceClass` that specified the `RDSInstanceClass` as
-  `mysql-standard` in the `app-project1-dev` namespace
-* Created a `MySQLInstance` claim in the `app-project1-dev1` namespace that
-  referenced `mysql-standard`
+* Created a `RDSInstanceClass` with configuration for an AWS RDS instance
+* Created a `MySQLInstance` claim that was scheduled to the `mysql-standard`
```
Comment (negz) [inline docs/services/aws-services-guide.md]: Nitpick: I've been saying "defaulted" when we use the annotation to distinguish it from using the class selector.

File: docs/services/gcp-services-guide.md [hunk 11]
```diff
@@ -59,40 +57,19 @@ export SUBNETWORK_NAME=default # the subnetwork that your GKE cluster lives in.
...
 ```
 
-3. Make sure `kubectl` is able to communicate with your GKE cluster
+2. Make sure `kubectl` is able to communicate with your GKE cluster
 ```bash
 kubectl cluster-info
 ```
-4. Make sure Helm is installed with permissions to work on `crossplane-system`
+
+3. Make sure Helm is installed with permissions to work on `crossplane-system`
```
Comment (negz) [inline docs/services/gcp-services-guide.md]: I feel like we could probably start the steps here - i.e. link off to some GKE docs and assume the user can figure out how to get GKE running and connect to it using `kubectl`.

File: docs/services/gcp-services-guide.md [hunk 12]
```diff
@@ -153,19 +130,20 @@ acquired from GCP:
     spec:
       credentialsSecretRef:
         name: gcp-provider-creds
+        namespace: crossplane-system
         key: credentials.json
       projectID: $PROJECT_ID
     EOF
 
-    kubectl -n $INFRA_NAMESPACE apply -f gcp-provider.yaml
+    kubectlapply -f gcp-provider.yaml
```
Comment (negz) [inline docs/services/gcp-services-guide.md]:
```suggestion
    kubectl apply -f gcp-provider.yaml
```

File: docs/services/gcp-services-guide.md [hunk 13]
```diff
@@ -153,19 +130,20 @@ acquired from GCP:
...
 * Verify GCP provider was successfully registered by the crossplane
 
   ```bash
-  kubectl -n $INFRA_NAMESPACE get providers.gcp.crossplane.io
-  kubectl -n $INFRA_NAMESPACE get secrets
+  kubectl get providers.gcp.crossplane.io
+  kubectl -n crossplane-system get secrets
   ```
 
 #### Cloud-Specific Resource Classes
```
Comment (negz) [inline docs/services/gcp-services-guide.md]: We can probably drop the "cloud-specific" terminology now that there's only one form of resource class.

File: docs/services/gcp-services-guide.md [hunk 14]
```diff
@@ -110,12 +87,11 @@ helm init --service-account=tiller
...
+To keep your resource configuration organized, start by creating a new
+directory:
+
 ```bash
-kubectl create namespace gcp-infra-dev
-export INFRA_NAMESPACE=gcp-infra-dev
+mkdir wordpress && cd $_
 ```
 
 #### GCP Provider
```
Comment (negz) [inline docs/services/gcp-services-guide.md]: Can we defer to https://crossplane.io/docs/master/cloud-providers/gcp/gcp-provider.html rather than re-documenting this? I believe that's what the AWS services guide did.

File: docs/services/gcp-services-guide.md [hunk 15]
```diff
@@ -328,13 +237,13 @@ You can omit `spec.classRef` if you want to use the default MySQLInstanceClass:
   kubectl -n app-project1-dev apply -f mysql-claim.yaml
```
Comment (negz) [inline docs/services/gcp-services-guide.md]:
```suggestion
  kubectl apply -f mysql-claim.yaml
```

File: docs/services/gcp-services-guide.md [hunk 16]
```diff
@@ -348,13 +257,13 @@ You can observe resource creation progression using the following:
```
Comment (negz) [inline docs/services/gcp-services-guide.md]:
```
NAME          STATUS   CLASS-KIND         CLASS-NAME       RESOURCE-KIND   RESOURCE-NAME               AGE
```

All claims have this style output these days, so the above example is not entirely accurate.

File: docs/services/gcp-services-guide.md [hunk 17]
```diff
@@ -443,15 +354,15 @@ use Crossplane to do it:
...
+  kubectl apply -f network.yaml
   ```
 
 * You can verify creation with the following command and output:
 
   *Command*
 
   ```bash
-  kubectl -n $INFRA_NAMESPACE get connection example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
+  kubectl get connection example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
```
Comment (negz) [inline docs/services/gcp-services-guide.md]: Should we just say you can `kubectl describe` it here as we do elsewhere? This is fancy but a bit intimidating. It's also probably more useful to have some context on what `FIRST_CONDITION` and `SECOND_CONDITION` are (plus there's three condition types now).

File: docs/services/gcp-services-guide.md [hunk 18]
```diff
@@ -472,21 +383,21 @@ variables. It should have been populated with our MySQL connection details after
...
+kubectl describe secret wordpressmysql
 ```
 
 *Output*
 ```bash
 Name:         wordpressmysql
-Namespace:    app-project1-dev
+Namespace:    default
 Labels:       <none>
 Annotations:  <none>
```
Comment (negz) [inline docs/services/gcp-services-guide.md]: Consistency nitpick: this will have the propagation annotations included in the AWS guide. Might be nice to either include or exclude them consistently.

File: docs/services/gcp-services-guide.md [hunk 19]
```diff
@@ -583,30 +494,19 @@ becomes available, then navigate to the address. You should see the following:
...
 ## Clean Up
 
-### Wordpress
-
-The `Service` and `Deployment` of Wordpress we installed can be removed with the following command:
+Because we put all of our configuration in a single directory, we can delete it all with this command:
 
 ```bash
-kubectl -n app-project1-dev delete -f wordpress.yaml
+kubectl delete -f wordpress/
```
Comment (negz) [inline docs/services/gcp-services-guide.md]:
```suggestion
kubectl delete -Rf wordpress/
```

File: docs/services/aws-services-guide.md [hunk 20]
```diff
@@ -506,212 +239,101 @@ that we configured earlier.
...
-  ```
+#### Resource Claims
 
-Once again, you are free to create more `MySQLInstanceClass` instances in this
-namespace to define more classes of service. For instance, if you created
-`mysql-aws-large` above, you may want to create a `MySQLInstanceClass` named
-`mysql-large` that references it. You may also choose to create MySQL resource
-classes for other non-AWS providers, and reference them for a class of service
-in the `app-project1-dev` namespace.
+Resource claims are used to create external resources by being scheduled to a
```
Comment (hasheddan) [inline docs/services/aws-services-guide.md]: @prasek per our conversation on Slack, I have added slight differences to how each services guide creates a claim that matches to the desired class. I think this helps demonstrate the power of scheduling / defaulting. See the `Resource Claim` sections in the other two guides for context.

File: docs/services/azure-services-guide.md [hunk 21]
```diff
@@ -13,11 +10,10 @@ Crossplane managed resources and the official Wordpress Docker image.
 
 1. [Pre-requisites](#pre-requisites)
 2. [Preparation](#preparation)
-3. [Set Up AKS Cluster](#set-up-aks-cluster)
-4. [Set Up Crossplane](#set-up-crossplane)
-5. [Install Wordpress](#install-wordpress)
-6. [Uninstall](#uninstall)
-7. [Conclusion and Next Steps](#conclusion-and-next-steps)
+3. [Set Up Crossplane](#set-up-crossplane)
```
Comment (prasek) [inline docs/services/azure-services-guide.md]: I like the simpler ToC, and shorter doc in general, it's a big improvement!

Suggest the following ToC:

1. Pre-requisites
1. Preparation
1. Set Up Crossplane
    1. Install Crossplane
    1. Cloud Provider
    1. Resource Classes
    1. Configure Managed Service Access (1)
1. Provision MySQL
   1. Resource Claim
   1. VNet Rule
1. Install Wordpress
1. Clean Up
1. Conclusion and Next Steps

(1) suggest moving the "Virtual Network with the Microsoft.Sql service endpoint enabled" into a very short section here (Configure Managed Service Access), so that step can be common across the guides.

I used a similar pattern in the manual steps section of these posts:
 - https://blog.crossplane.io/gcp-secure-connectivity-for-gke-cloudsql/
 - https://blog.crossplane.io/aws-secure-connectivity-for-eks-rds/
 - https://blog.crossplane.io/azure-secure-connectivity-for-aks-azure-db/

File: docs/services/gcp-services-guide.md [hunk 22]
```diff
@@ -16,10 +16,8 @@ indent: true
      2. [Crossplane Steps](#crossplane-steps)
```
Comment (prasek) [inline docs/services/gcp-services-guide.md]: Really like the cleaned up flow for the Azure Service Guide. Suggest following that here in the GCP Service Guide as much as possible.

Suggest the following ToC:

1. Pre-requisites
1. Preparation
1. Set Up Crossplane
    1. Install
    1. Cloud Provider
    1. Resource Classes
    1. Configure Managed Service Access (note: you only need one private connection for all services)
1. Provision MySQL
   1. Resource Claim
1. Install Wordpress
1. Clean Up
1. Conclusion and Next Steps

File: docs/services/azure-services-guide.md [hunk 23]
```diff
@@ -35,58 +31,31 @@ local machine.
 ## Preparation
 
 This guide assumes that you have setup the Azure CLI and are logged in to your
-desired account.
+desired account. It also assumes that you have an existing AKS cluster in a
+Virtual Network with the `Microsoft.Sql` [service endpoint] enabled. Make sure
```
Comment (prasek) [inline docs/services/azure-services-guide.md]: see ToC comment above

File: docs/services/gcp-services-guide.md (hunk 11 above)
Comment (prasek) [reply docs/services/gcp-services-guide.md]: Generally agree that shorter is better, though the short verification steps help orient and make sure they're ready for the next step. Would be fine either way IMO.

File: docs/services/gcp-services-guide.md (hunk 13 above)
Comment (prasek) [inline docs/services/gcp-services-guide.md]: remove "Cloud-Specific"

File: docs/services/gcp-services-guide.md [hunk 24]
```diff
@@ -220,92 +200,20 @@ You are free to create more GCP `CloudsqlInstanceClass` instances to define more
...
-
 #### Resource Claims
 
-Resource claims are used to create external resources by referencing a class of
-service in the claim's namespace. When a claim is created, Crossplane uses the
-referenced portable class to find a cloud-specific resource class to use as the
-configuration for the external resource. We need to create a claim to
-provision the MySQL database we will use in GCP.
+Resource claims are used to create external resources by being scheduled to a
+resource class and creating new managed resource or binding to an existing
```
Comment (prasek) [inline docs/services/gcp-services-guide.md]: Suggest rewording this section for readability by a new user.

e.g.  Resource claims are used for dynamic provisioning of a managed resource (like a MySQL instance) by matching the claim to a resource class. This can be done in several ways: (a) rely on the default class marked  `resourceclass.crossplane.io/is-default-class: "true"`, (b) use a `claim.spec.classRef` to a specific class, or (c) match on class labels using a `claim.spec.classSelector`.

Note: claims may also be used in [static provisioning](link to quickstart guide) with a reference to an existing managed resource.

File: docs/services/azure-services-guide.md [hunk 25]
```diff
@@ -262,124 +209,35 @@ You are free to create more Azure `SQLServerClass` instances to define more
...
-```
-
 #### Resource Claims
 
-Resource claims are used to create external resources by referencing a class of
-service in the claim's namespace. When a claim is created, Crossplane uses the
-referenced portable class to find a cloud-specific resource class to use as the
-configuration for the external resource. We need a to create a claim to
-provision the MySQL database we will use with Azure.
+Resource claims are used to create external resources by being scheduled to a
```
Comment (prasek) [inline docs/services/azure-services-guide.md]: Like showing the options. Suggest rewording this section for readability by a new user.

e.g.  Resource claims are used for dynamic provisioning of a managed resource (like a MySQL instance) by matching the claim to a resource class. This can be done in several ways: (a) rely on the default class marked  `resourceclass.crossplane.io/is-default-class: "true"`, (b) use a `claim.spec.classRef` to a specific class, or (c) match on class labels using a `claim.spec.classSelector`.

Note: claims may also be used in [static provisioning](link to quickstart guide) with a reference to an existing managed resource.

File: docs/services/azure-services-guide.md [hunk 26]
```diff
@@ -221,10 +166,12 @@ apiVersion: database.azure.crossplane.io/v1alpha2
 kind: SQLServerClass
 metadata:
   name: azure-mysql-standard
-  namespace: azure-infra-dev
+  labels:
+    app: wordpress
```
Comment (prasek) [inline docs/services/azure-services-guide.md]: the use of `app: wordpress` implies a class will be created for each app stack which seems at odds with a name of `azure-mysql-standard`. Would suggest using a label like `kind: mysql-standard` or something the reflects the concept of a portable class -- just expressed as a label here.

File: docs/services/aws-services-guide.md [hunk 27]
```diff
@@ -14,11 +11,9 @@ Crossplane managed resources and the official Wordpress Docker image.
 
 1. [Pre-requisites](#pre-requisites)
 1. [Preparation](#preparation)
-1. [Set Up an EKS Cluster](#set-up-an-eks-cluster)
-1. [Set Up RDS Configurations](#set-up-rds-configurations)
-1. [Set Up Crossplane](#set-up-crossplane)
+1. [Create Configuration](#create-configuration)
```
Comment (prasek) [inline docs/services/aws-services-guide.md]: I like the simpler ToC, and shorter doc in general, it's a big improvement!

Suggest the following ToC:

1. Pre-requisites
1. Preparation
1. Set Up Crossplane
    1. Install
    1. Cloud Provider
    1. Resource Classes
    1. Configure Managed Service Access (DBSubnetGroup, SecurityGroup)
1. Provision MySQL
   1. Resource Claim
1. Install Wordpress
1. Clean Up
1. Conclusion and Next Steps

File: docs/services/aws-services-guide.md (hunk 2 above)
Comment (prasek) [reply docs/services/aws-services-guide.md]: https://github.com/hasheddan/crossplane/pull/3

File: docs/services/aws-services-guide.md (hunk 20 above)
Comment (prasek) [reply docs/services/aws-services-guide.md]: @hasheddan thanks, that definitely helps. Suggest rewording this section for readability by a new user.

e.g.  Resource claims are used for dynamic provisioning of a managed resource (like a MySQL instance) by matching the claim to a resource class. This can be done in several ways: (a) rely on the default class marked  `resourceclass.crossplane.io/is-default-class: "true"`, (b) use a `claim.spec.classRef` to a specific class, or (c) match on class labels using a `claim.spec.classSelector`.

Note: claims may also be used in [static provisioning](link to quickstart guide) with a reference to an existing managed resource.

File: docs/services/aws-services-guide.md (hunk 4 above)
Comment (hasheddan) [reply docs/services/aws-services-guide.md]: hmm I definitely see your point here. However, I do think it is much faster going through the guide if you can just copy paste these commands. Long-term I think it would be cool to have text-boxes at the top where you could enter values and then all of the config is adjusted with your values. In the meantime, I would prefer to stick with this `cat` convention, but am more than happy to update if we decide to move away from it.

File: docs/services/aws-services-guide.md (hunk 7 above)
Comment (hasheddan) [reply docs/services/aws-services-guide.md]: All files are top-level in `wordpress/` to no need to delete recursively.

File: docs/services/gcp-services-guide.md (hunk 19 above)
Comment (hasheddan) [reply docs/services/gcp-services-guide.md]: https://github.com/crossplaneio/crossplane/pull/975#discussion_r339286496

File: docs/services/aws-services-guide.md (hunk 8 above)
Comment (hasheddan) [reply docs/services/aws-services-guide.md]: Nope, good catch!

File: docs/services/gcp-services-guide.md (hunk 17 above)
Comment (hasheddan) [reply docs/services/gcp-services-guide.md]: Agreed that this should be changed. I am likely going to want to update output for a number of these commands when we go through and do final testing (after all code changes are made including `v1alpha3` stuff), so I may leave as-is for now.

File: docs/services/azure-services-guide.md (hunk 25 above)
Comment (hasheddan) [reply docs/services/azure-services-guide.md]: Went with a link to the `concepts.md` section on dynamic and static provisioning. Happy to update if you strongly prefer link to quickstart.

File: docs/services/gcp-services-guide.md [hunk 28]
```diff
@@ -56,123 +49,38 @@ export NETWORK_NAME=default # the network that your GKE cluster lives in.
...
-
-  ```bash
-  kubectl -n $INFRA_NAMESPACE get providers.gcp.crossplane.io
-  kubectl -n $INFRA_NAMESPACE get secrets
-  ```
-
-#### Cloud-Specific Resource Classes
+### Resource Classes
 
 Cloud-specific resource classes are used to define a reusable configuration for
```
Comment (prasek) [inline docs/services/gcp-services-guide.md]: remove cloud-specific?

Comment (hasheddan) [reply docs/services/gcp-services-guide.md]: oh shoot missed that one :sweat_smile:

File: docs/services/aws-services-guide.md (hunk 4 above)
Comment (negz) [reply docs/services/aws-services-guide.md]: I'm really not a fan, but I know this was a decision that was consciously made going into 0.3 and it doesn't _have_ to be fixed now. I'd like us to reassess whether this really makes our docs friendlier/easier though in future.

Comment (hasheddan) [reply docs/services/aws-services-guide.md]: If it is something we are definitely going to change, I would prefer doing so now rather than putting off. One nice thing about it is that you end up with all of your config in a single directory that you can `kubectl delete -f dir/` at the end if you want. Obviously you could do this anyway by just instructing the user to create files in directory, but it feels a little more definite this way.

File: docs/services/azure-services-guide.md [hunk 29]
```diff
@@ -4,6 +4,7 @@ toc: true
 weight: 440
 indent: true
 ---
+
 # Deploying Wordpress in Azure
```
Comment (prasek) [inline docs/services/azure-services-guide.md]:
```suggestion
# Azure Services Guide
```

Comment (hasheddan) [reply docs/services/azure-services-guide.md]: Updated here and for AWS!

File: docs/services/aws-services-guide.md [hunk 30]
```diff
@@ -14,388 +14,55 @@ Crossplane managed resources and the official Wordpress Docker image.
```
Comment (prasek) [inline docs/services/aws-services-guide.md]: Suggest changing the main heading to "AWS Services Guide" to more close links to this as the AWS services guide, to more closely match the ToC title, and to be consistent with the GCP Services Guide.

Comment (hasheddan) [reply docs/services/aws-services-guide.md]: This was updated in most recent commit. Reference here is to commit prior :+1:
